src/schemas/user.entity.ts

```diff
@@ -1,16 +1,37 @@
 import {CartEntity, CartSchema} from "./cart.entity";
 import mongoose, {Schema, Document} from "mongoose";
-
-export interface UserEntity {
-  id: string; // uuid
-}
+import { v4 as uuidv4 } from "uuid";
```

Comment:
You can use `randomUUID` from node's `crypto` modue. Then you won't need third-party dependence here.